.gitignore

```diff
@@ -87,3 +87,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# Jetbrains project files
+.idea/
```

Comment:
You should add your IDE files to your custom gitignore. To to EVERY project you may come to work with.